components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/contract/config/SslConfiguration.java

```diff
@@ -236,26 +240,29 @@ private SSLConfig getSSLConfigForListener() {
     }
 
     private SSLConfig getSSLConfigForSender() {
-        if ((sslConfig.getTrustStore() == null || sslConfig.getTrustStorePass() == null) && (
-                sslConfig.getClientTrustCertificates() == null)) {
-            throw new IllegalArgumentException("TrustStoreFile or TrustStorePassword not defined for HTTPS/WS scheme");
-        }
-        if (sslConfig.getTrustStore() != null) {
-            if (!sslConfig.getTrustStore().exists()) {
-                throw new IllegalArgumentException("TrustStore File " + sslConfig.getTrustStore() + " not found");
+        if (!sslConfig.isDisableSsl()) {
```

Comment:
Shall we check the inverse of this condition and return?
```suggestion
        if (sslConfig.isDisableSsl()) {
               return sslConfig;
        }
```